client/my-sites/checkout/checkout/composite-checkout.js

```diff
@@ -155,6 +162,89 @@ export default function CompositeCheckout( {
 		]
 	);
 
+	function getTermText( term ) {
+		switch ( term ) {
+			case TERM_BIENNIALLY:
+				return translate( 'Two years', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **8** times:
`translate( 'Two years' )` *ES Score: 14*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/plan-length-picker-to-composite-checkout)


:information_source: This string already exists without a **context**. Only add a context if the meaning of the string is very specific.